packages/plugin-fontawesome/src/FontAwesomeDomObjectRenderer.ts

```diff
@@ -20,11 +20,28 @@ export class FontAwesomeDomObjectRenderer extends NodeRenderer<DomObject> {
         node: FontAwesomeNode,
         worker: RenderingEngineWorker<DomObject>,
     ): Promise<DomObject> {
+        const select = (ev: Event) => {
+            ev.preventDefault();
+            const selectFontAwesome = () => {
+                this.engine.editor.selection.select(node, node);
+            };
```

Comment:
Why do you need to create a variable function? You only use it once.

Reply:
To be able to have the custom command named in the JW dev tool (best practice recommended by @Gorash )

Reply:
ah yes I forgot